x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwtRealm.java

```diff
@@ -61,8 +61,8 @@ record JwksAlgs(List<JWK> jwks, List<String> algs) {}
     final List<String> allowedAudiences;
     final String jwkSetPath;
     final CloseableHttpAsyncClient httpClient;
-    final JwtRealm.JwksAlgs jwksAlgsHmac;
-    JwtRealm.JwksAlgs jwksAlgsPkc; // reloadable
+    JwtRealm.JwksAlgs jwksAlgsHmac;
```

Comment:
Reloadable is in a different milestone. Initially only PKC JWKSet was going to be reloadable, but I asked Tim and I think we agreed to make HMAC keys reloadable in future too. He said we support reloading for a few ES keystore settings, but we need to do more of that, so I think it makes sense to remove the final qualifier.

Reply:
If it's reloadable in the future we can drop `final` then. There's no value in doing that now (we don't need to guess on what we will implement in the future & how it would be implemented)

Reply:
I will make it final. It means I have to change the signature of two methods to return those records so I can set final in the constructor.

Reply:
Resolved.